examples/java/spring-boot/README.md

```diff
@@ -0,0 +1,16 @@
+# Spring Boot application
+
+This is a Spring Boot application built from the [schemas for the federated model](https://github.com/co-cddo/federated-api-model/tree/main/schemas) in the top level of this repository.
+
+The app is deployed in a sandbox environment on PaaS, and has an [/apis endpoint](https://federated-api-model-spring-boot-sandbox.london.cloudapps.digital/apis) for returning metadata about other APIs.
```

Comment:
I would recommend pointing to / making clear there are OpenAPI spec(s) and that it can be used to interact with.

Then maybe add a "for example, you can use the `/apis` endpoint"